src/__tests__/store/reducers/healthCheck.reducer.test.js

```diff
@@ -7,77 +7,164 @@ import {
     FETCH_HEALTHCHECK_SUCCESS,
     DISCARD_PAGINATION,
     SET_PAGINATION_PAGE,
+    SET_SORTING_FILTER,
+    SET_ASCEND_FILTER,
 } from '../../../store/constants';
 
 import { checkStatus } from '../../../store/reducers/healthcheck.reducer';
+import touchedReducerProps from '../../../helpers/touchedReducerProperties';
 
 const getRandomString = () => Math.floor(Math.random() * 10000).toString(16);
 
 describe('healthCheckRducer', () => {
-    it('returns the initial state by default', () => {
-        const result = healthCheckRducer(initialState, {});
+    describe('Default', () => {
+        it('returns the initial state by default', () => {
+            const result = healthCheckRducer(initialState, {});
 
-        expect(result).toEqual(initialState);
+            expect(result).toEqual(initialState);
+        });
     });
 
-    it('returns isFetching', () => {
-        const result = healthCheckRducer(initialState, { type: FETCH_HEALTHCHECK_LIST_START });
+    describe('FETCH_HEALTHCHECK_LIST_START', () => {
+        it('returns isFetching', () => {
+            const result = healthCheckRducer({}, { type: FETCH_HEALTHCHECK_LIST_START });
 
-        expect(result.isFetching).toBe(true);
+            expect(result.isFetching).toBe(true);
+            expect(touchedReducerProps(result)).toBe(1);
+        });
     });
 
-    it('returns isFetching', () => {
-        const result = healthCheckRducer(initialState, { type: FETCH_HEALTHCHECK_START });
+    describe('FETCH_HEALTHCHECK_START', () => {
```

Comment:
sorry for the dumb question, is this a new style of describing tests?

Reply:
that is just for the more prettier outcome:

![image](https://user-images.githubusercontent.com/8372070/31434621-2140bcf4-ae7d-11e7-8958-374400c1ea08.png)

Reply:
😮 in fact it is better to read on the console

Reply:
But this is snapshot of the `console`